extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,459 @@
+					<option value="CR">Croatian</option>
+					<option value="CZ">Czech</option>
+					<option value="DK">Danish</option>
+					<option value="DV">Dhivehi</option>
+					<option value="NL">Dutch</option>
+					<option value="EN">English</option>
+					<option value="EO">Esperanto</option>
+					<option value="ET">Estonian</option>
+					<option value="FA">Farsi</option>
+					<option value="FI">Finnish</option>
+					<option value="FR">French</option>
+					<option value="FC">French Canadian</option>
+					<option value="GZ">Galician</option>
+					<option value="DL">German</option>
+					<option value="KA">Georgian</option>
+					<option value="GR">Greek</option>
+					<option value="GU">Gujarati</option>
+					<option value="HT">Haitian Creole</option>
+					<option value="IL">Hebrew</option>
+					<option value="HI">Hindi</option>
+					<option value="HU">Hungarian</option>
+					<option value="IS">Icelandic</option>
+					<option value="IO">Ido</option>
+					<option value="ID">Indonesian</option>
+					<option value="IR">Irish Gaelic</option>
+					<option value="IT">Italian</option>
+					<option value="JP">Japanese</option>
+					<option value="JW">Javanese</option>
+					<option value="KM">Khmer</option>
+					<option value="KR">Korean</option>
+					<option value="KU">Kurdish</option>
+					<option value="LA">Latin</option>
+					<option value="LV">Latvian</option>
+					<option value="LT">Lithuanian</option>
+					<option value="ND">Low German</option>
+					<option value="MK">Macedonian</option>
+					<option value="MT">Maltese</option>
+					<option value="GM">Mandinka</option>
+					<option value="MI">Maori</option>
+					<option value="MR">Marathi</option>
+					<option value="MN">Mongolian</option>
+					<option value="NO">Norwegian</option>
+					<option value="OC">Occitan</option>
+					<option value="PS">Pashto</option>
+					<option value="GN">Plautdietsch</option>
+					<option value="PL">Polish</option>
+					<option value="BR">Portuguese</option>
+					<option value="PA">Punjabi</option>
+					<option value="RO">Romanian</option>
+					<option value="RU">Russian</option>
+					<option value="SR">Serbian</option>
+					<option value="SK">Slovak</option>
+					<option value="SL">Slovenian</option>
+					<option value="SP">Spanish</option>
+					<option value="SI">Swahili</option>
+					<option value="SW">Swedish</option>
+					<option value="CH">Swiss</option>
+					<option value="TL">Tagalog</option>
+					<option value="TT">Tatarish</option>
+					<option value="TH">Thai</option>
+					<option value="TR">Turkish</option>
+					<option value="TK">Turkmen</option>
+					<option value="UA">Ukrainian</option>
+					<option value="UZ">Uzbek</option>
+					<option value="VU">Vietnamese</option>
+					<option value="CY">Welsh</option>
+					<option value="SN">Wolof</option>
+					<option value="JI">Yiddish - transliterated</option>
+					<option value="YI">Yiddish - unicode</option>
+				</options>
+				<default>EN</default>
```

Comment:
They are part of the ConfigDescriptions which are not directly referenced by the configuration object, therefore I'm not aware of an "official" way to get these.
May I ask for the use-case? Because normally it's the other way round: The XML files simply describe what the ThingHandler expects or takes for granted.